OAK_Marketplace_Emotion_Recognition/utils/helpers.py

```diff
@@ -0,0 +1,16 @@
+import skvideo.io
+import os
+
+FFMPEG_OUTPUT_DICT = {
+    "-vcodec": "libx265",
+    "-vf": "format=yuv420p",
+    "-movflags": "+faststart",
+}
+
+
+def save_video(visualization_results):
+    writer = skvideo.io.FFmpegWriter("inference_results.mp4", outputdict=FFMPEG_OUTPUT_DICT,
```

Comment:
Please, mention that ffmpeg must be installed. Create a section with requirements. Also, mention what Python version is used. For more details, visit this README - https://github.com/opencv-ai/opencv-blog/tree/main/PyTorch_ONNX_OpenVINO#requirements